src/html/partials/behind-the-scenes.html

```diff
@@ -1,4 +1,57 @@
 <section class="behind-scenes-section">
+
   <h2 class="sm-title">Behind The Scenes</h2>
-  <p class="bg-intro">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
-</section>
+  <p class="lg-text">While Florence is only a concept, here are some things that will be helpful in making her come alive.</p>
+
+  <!-- <div class="behind-scenes-section__florence">
+    <div class="behind-scenes-section__florence__book"></div>
+    <div class="behind-scenes-section__florence__gear"></div>
+    <div class="behind-scenes-section__florence__comment"></div>
+    <div class="behind-scenes-section__florence__heart"></div>
+    <div class="behind-scenes-section__florence__link"></div>
```

Comment:
why are these commented out?

Reply:
I believe these might be for the mobile-only icons. Mobile has two sets of icons, desktop has one.